cpp/tests/test_abm_person.cpp

```diff
@@ -321,12 +329,58 @@ TEST(Person, rng)
     mio::abm::Location loc(mio::abm::LocationType::Home, 0);
     auto p = mio::abm::Person(rng, loc, age_group_35_to_59, 13);
 
-    ASSERT_EQ(p.get_rng_counter(), mio::Counter<uint32_t>(0));
+    EXPECT_EQ(p.get_rng_counter(), mio::Counter<uint32_t>(0));
 
     auto p_rng = mio::abm::Person::RandomNumberGenerator(rng, p);
-    ASSERT_EQ(p_rng.get_counter(), mio::rng_totalsequence_counter<uint64_t>(13, mio::Counter<uint32_t>{0}));
+    EXPECT_EQ(p_rng.get_counter(), mio::rng_totalsequence_counter<uint64_t>(13, mio::Counter<uint32_t>{0}));
 
     p_rng();
-    ASSERT_EQ(p.get_rng_counter(), mio::Counter<uint32_t>(1));
-    ASSERT_EQ(p_rng.get_counter(), mio::rng_totalsequence_counter<uint64_t>(13, mio::Counter<uint32_t>{1}));
+    EXPECT_EQ(p.get_rng_counter(), mio::Counter<uint32_t>(1));
+    EXPECT_EQ(p_rng.get_counter(), mio::rng_totalsequence_counter<uint64_t>(13, mio::Counter<uint32_t>{1}));
+}
+
+TEST(TestPerson, isCompliant)
+{
+    using testing::Return;
+
+    // Initialize the random number generator
+    auto rng = mio::RandomNumberGenerator();
+
+    // Create locations
+    mio::abm::Location home(mio::abm::LocationType::Home, 0, num_age_groups);
```

Comment:
not needed

Reply:
But do we need a location to use the function `make_test_person`?

Reply:
Ah, yes, sorry!